ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestSuiteRunner.scala

```diff
@@ -131,7 +138,19 @@ final class LedgerTestSuiteRunner(
     Source(tests)
       .mapAsyncUnordered(concurrentTestRuns) {
         case ((suite, test), index) =>
-          run(test, suite.session).map(testResult => (suite, test, testResult) -> index)
+          val matchName: String = suite.name + ":" + test.shortIdentifier
```

Comment:
If this is the new canonical representation for test case names, it could make sense to define it in a single place. A simple method will do.

Reply:
Right, but where? The test case as currently defined does not know which suite it's part of. Also thinking more about it I'm quite unhappy with the state in which this PR leaves the code, as there is an annoying (and error-prone) distinction between the test suite prefix in the test names, used by the regex, and the test suite names in the map, used by the `--include` and `--exclude` options.

I'll mull this over a bit longer.